src/redist/targets/Crossgen.targets

```diff
@@ -51,13 +51,18 @@
       <RemainingFiles Remove="$(SdkOutputDirectory)**\Microsoft.TestPlatform.Extensions.EventLogCollector.dll" />
 
       <!-- Add back the .NET Core assemblies in the Sdks folder -->
-      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk\tools\netcoreapp*\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk\tools\net6.0\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.BlazorWebAssembly\tools\net6.0\**\*" />
       <RemainingFiles Include="$(SdkOutputDirectory)Sdks\NuGet.Build.Tasks.Pack\CoreCLR\**\*" />
-      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Razor\tasks\netcoreapp*\*" />
-      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Razor\tools\netcoreapp*\**\*" />
+      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Razor\tasks\net6.0\**\*" />
```

Comment:
Would it help if we put these in a netcoreapp directory? That way we wouldn't have to worry about updating these every release. The TFM version isn't really useful to the SDK

/cc @captainsafia

Reply:
Could be worthwhile from a maintenance perspective.  I suspect that we're going to need to do something to catch new additions to the SDK (likely a test that looks for new non-crossgen'd assemblies), and so it may not be immediately necessary, but if this validation becomes costly or difficult, then renaming the directory could be a good way to ensure that things don't break going forward.

Reply:
It's not incredibly difficult to fix up the SDK to produce fixed paths, but I think adding a check would be a better approach.